### PR TITLE
Remove Exports Flags

### PR DESCRIPTION
**Pull request by biyeun**
- Flips over to new views
- Does not delete the old DataInterfaceDispatcher reports because those are currently tied into permissions. Swapping permissions should come in a different PR and be QA'd separately

@benrudolph 
@czue 
@dannyroberts

### CODE REVIEW

**Comment by biyeun** [comment]
@dimagi/product

File: corehq/apps/hqwebapp/models.py [hunk 1]
```diff
@@ -720,6 +715,21 @@ def sidebar_items(self):
 
         return items
 
+    @property
+    def dropdown_items(self):
+        if not self.can_edit_commcare_data:
+            return []
+        from corehq.apps.export.views import (
+                FormExportListView,
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/models.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/hqwebapp/models.py [hunk 2]
```diff
@@ -720,6 +715,21 @@ def sidebar_items(self):
...
+                CaseExportListView,
+        )
+        return [
+            dropdown_dict(FormExportListView.page_title, url=reverse(FormExportListView.urlname, args=(self.domain,))),
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/models.py]: E501 line too long (119 > 115 characters)

File: corehq/apps/hqwebapp/models.py [hunk 3]
```diff
@@ -720,6 +715,21 @@ def sidebar_items(self):
...
+            dropdown_dict(CaseExportListView.page_title, url=reverse(CaseExportListView.urlname, args=(self.domain,))),
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/models.py]: E501 line too long (119 > 115 characters)

**Comment by biyeun** [comment]
created a case for removing `DataInterfaceDispatcher` http://manage.dimagi.com/default.asp?188217 @dimagi/product we should discuss prioritizing this at some point, can also chat with Lucina about when to prioritize.

**Comment by biyeun** [comment]
the above shouldn't affect the new ui. The old ui + code will still unfortunately be in the codebase adding debt until this is cleared

**Comment by czue** [comment]
:balloon: :balloon: :balloon: 

I'm surprised there isn't more code to delete beyond `DataInterfaceDispatcher`. is there a separate chunk of stuff that still has dependencies?

**Comment by dannyroberts** [comment]
I was curious about this too. But @biyeun it sounds like

> The old ui + code will still unfortunately be in the codebase adding debt until this is cleared

refers both to the `DataInterfaceDispatcher` code you can't delete and to the other code that we can but just haven't gotten around to deleting?

**Comment by biyeun** [comment]
@dannyroberts the reason why I didn't delete it is that DataInterfaceDispatcher is tied into exports and de-id exports permissions. So we would have to change how permissions are handled and migrate existing permissions. I didn't want to change it here because this would require a little more thorough QA.

**Comment by dannyroberts** [comment]
ok thanks, sounds good

**Pull request merged**